app/views/admin/articles/_form.html.erb

```diff
@@ -157,8 +157,19 @@
     <div class="row">
       <div class="col-xs-12 col-sm-6">
         <div class="form-group">
-          <%= f.label :image, "Header Image" %>
-          <%= f.text_field :image, class: "form-control input-lg" %>
+          <p class="help-block">The article's header image is currently set to
+            <code>
+              <%= @article.article_image.url || @article.image %>
+            </code>
+          </p>
+          <%= f.label :article_image, "Header Image Upload" %>
+          <%= f.file_field :article_image, class: "form-control" %>
```

Comment:
Since the `store_dir` that I want needs to know what date the article will be published on, can you hide this part of the form on /new and only show it on /edit?